src/js/blocks/button/block.js

```diff
@@ -142,22 +138,22 @@ registerBlockType( 'uri-cl/button', {
 						<PanelRow>
 							<BaseControl
 								label={ __( 'Button Style' ) }
+								id="button-style"
 							>
 								<ButtonGroup aria-label={ __( 'Button Style' ) }>
 									{ [ 'default', 'prominent', 'disabled' ].map( ( value ) => {
-
 										const capitalizedValue = value.charAt( 0 ).toUpperCase() + value.slice( 1 );
 										const key = ( 'default' === value ) ? '' : value;
-										const style = ( undefined == attributes.style ) ? '' : attributes.style;
-										const isSelected = ( key === style );
+										const style = ( undefined === attributes.style ) ? '' : attributes.style;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4811013)